src/mail/dto/mail-verify.dto.ts

```diff
@@ -0,0 +1,10 @@
+import { IsString, Length } from 'class-validator'
+
+export class MailVerifyDto {
```

Comment:
The dto is only used as an interface. (Or am I wrong?) You have to call the ValidationPipe to valdiate data.

Reply:
yeah you are right
I will remove the decorators as the data does not have to be sanitized anyway. It cannot be malicious

Reply:
Maybe we should update their type to `interface` or `type`. Also shall we move them to a interfaces / types folder?